runtime-sdk/src/storage/mkvs.rs

```diff
@@ -21,6 +21,13 @@ impl<M: mkvs::MKVS> MKVSStore<M> {
     fn create_ctx(&self) -> Context {
         Context::create_child(&self.ctx)
     }
+
+    // TODO: fix the type in oasis-core.
```

Comment:
What is this about? Let's file an issue in Oasis Core and link it here.

Reply:
Oh it's clippy complaining about to replacing the argument `&Vec` with a slice. I actually fixed it for now by not using a reference for `prefetch_prefixes` in oasis-sdk.

Can open an issue in oasis-core for this, but actually i think that since clippy is not setup in oasis-core, there might be bunch of lint fixes needed - should just setup clippy in oasis-core and address all of the issues found, so i'll open issue for that. Probably should be done only after cleaning up/removing all of the deprecated stuff.

Reply:
Yeah let's open an issue to setup clippy in Core.

Reply:
https://github.com/oasisprotocol/oasis-core/issues/4170